filament/src/VertexBuffer.cpp

```diff
@@ -123,6 +123,19 @@ VertexBuffer* VertexBuffer::Builder::build(Engine& engine) {
         return nullptr;
     }
 
+    auto const& declaredAttributes = mImpl->mDeclaredAttributes;
+    auto const& attributes = mImpl->mAttributes;
+    utils::bitset32 mAttributedBuffers;
```

Comment:
It's kind of weird to have a local variable with the `m` prefix -- was it the intention?

Reply:
Woops, fixed.